lib/Raven/Client.php

```diff
@@ -979,13 +672,43 @@ private function isEncodingCompressed()
         return 'gzip' === $this->config->getEncoding();
     }
 
-    private function autoloadRavenStacktrace()
+    /**
+     * Seeds the middleware stack with the first callable.
+     */
+    private function seedMiddlewareStack()
```

Comment:
Can't we do it in the constructor once and for all?

Reply:
Indeed, original code for the middleware part was taken from a PHP `trait` available to Slim framework users and meant to be reusable, but there is no reason we have the same requirements here 👍